From 04f6f3a1d89c875bd5a8bf0d7309efd5cafd7efb Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Thu, 23 Aug 2018 06:46:44 +0300
Subject: [PATCH] Makefile: add .DEFAULT_GOAL content of HL_VARS_FILE

Now one can know value of HL_DEV and HL_TARGET
---

diff --git a/Makefile b/Makefile
index 019ba03..29abe5a 100644
--- a/Makefile
+++ b/Makefile
@@ -1,3 +1,17 @@
+export BR2_EXTERNAL := $(CURDIR)
+OUTPUT_DIR := $(BR2_EXTERNAL)/output
+$(OUTPUT_DIR):
+	@mkdir -p $@
+
+Makefile:;
+%:
+	@$(MAKE) -C ../buildroot O=$(OUTPUT_DIR) $@
+
+HL_VARS_FILE := $(OUTPUT_DIR)/.hl-vars.mk
+ifneq ($(realpath $(HL_VARS_FILE)),)
+include $(HL_VARS_FILE)
+endif
+
 .DEFAULT_GOAL := hl-help
 hl-help:
 	@echo 'Cleaning:'
@@ -39,19 +53,10 @@
 	@echo '                             override <board> dafault IP'
 	@echo '                           set PCIE_HOST_TBOT_IP=<PCIe host IP> to'
 	@echo '                             override <board> dafault IP'
-
-export BR2_EXTERNAL := $(CURDIR)
-OUTPUT_DIR := $(BR2_EXTERNAL)/output
-$(OUTPUT_DIR):
-	@mkdir -p $@
-
-Makefile:;
-%:
-	@$(MAKE) -C ../buildroot O=$(OUTPUT_DIR) $@
-
-HL_VARS_FILE := $(OUTPUT_DIR)/.hl-vars.mk
 ifneq ($(realpath $(HL_VARS_FILE)),)
-include $(HL_VARS_FILE)
+	@echo 'Current Variables:'
+	@echo '  HL_DEV = $(HL_DEV)'
+	@echo '  HL_TARGET = $(HL_TARGET)'
 endif
 
 .PHONY: hl-check-release
